doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -197,6 +210,34 @@ with a ``.ipynb`` file.
 
 .. _MyST-NB: https://myst-nb.readthedocs.io/
 
+.. _interactive-docs:
+
+Interactive examples in docstrings
```

Comment:
Why move this? Do contributors have to think about this unless they're rendering locally?

Reply:
I think having an explanation of these directives is important. Even if just updating the docstrings they are going to bump into the directives and may be confused since they are not yet widely used in other projects etc. But I think it's a good point that I didn't include an explanation of the directive here, so I'll do that. I'm also happy to move it back if you feel it's more appropriate.

Reply:
Sorry, maybe I am missing something. Can you point to examples of these directives? Do we have to use them manually to make individual examples interactive? I thought orginally the idea was that all docstring examples would be rendered as interactive online (and presumably it is just turned off right now)?

Reply:
I am clarifying with the jupyterlite-sphinx folks and will update here accordingly.

Reply:
Thanks. I think the assumption here should be that the contributor wants all docstring examples to have the button to make it interactive (if that is the SciPy default). They probably don't need to do anything manually to make that happen in the real, rendered documentation, but if there's something they need to do to make it happen locally, that's worth mentioning here.

I think other information, like how to *disable* the button ("To hide the buttons that turn docstring examples into embedded notebooks") might belong in the core developer guide, but I don't know that all contributors need it.

Reply:
My fear with moving some of this documentation to the core devs guide is that is will create discoverability problems - do you think this would be findable by searching the docs?

Reply:
Who needs that information, though?

Reply:
Ok, that makes sense. I will try to find a place for that in the core devs guide. To be clear this information is also kind of in flux right now as changes are made to jupyterlite-sphinx, so they are not permanent.